api/server/server.go

```diff
@@ -52,15 +55,96 @@ func (s *Server) Run() error {
 	router.Use(middlewareLogger(s.logger))
 	router.Use(middlewareNodeVersion)
 
+	// @Summary Get node identity information
+	// @Description Returns the identity information of the SSV node
+	// @Tags Node
+	// @Produce json
+	// @Success 200 {object} handlers.NodeIdentityResponse
+	// @Router /v1/node/identity [get]
 	router.Get("/v1/node/identity", api.Handler(s.node.Identity))
+
+	// @Summary Get connected peers
+	// @Description Returns the list of peers connected to the SSV node
+	// @Tags Node
+	// @Produce json
+	// @Success 200 {object} handlers.PeersResponse
+	// @Router /v1/node/peers [get]
 	router.Get("/v1/node/peers", api.Handler(s.node.Peers))
+
+	// @Summary Get subscribed topics
+	// @Description Returns the list of topics the SSV node is subscribed to
+	// @Tags Node
+	// @Produce json
+	// @Success 200 {object} handlers.TopicsResponse
+	// @Router /v1/node/topics [get]
 	router.Get("/v1/node/topics", api.Handler(s.node.Topics))
+
+	// @Summary Get node health status
+	// @Description Returns the health status of the SSV node
+	// @Tags Node
+	// @Produce json
+	// @Success 200 {object} handlers.HealthResponse
+	// @Router /v1/node/health [get]
 	router.Get("/v1/node/health", api.Handler(s.node.Health))
+
+	// @Summary Get validators
+	// @Description Returns the list of validators managed by the SSV node
+	// @Tags Validators
+	// @Produce json
+	// @Success 200 {object} handlers.ValidatorsResponse
+	// @Router /v1/validators [get]
 	router.Get("/v1/validators", api.Handler(s.validators.List))
 
 	// We kept both GET and POST methods to ensure compatibility and avoid breaking changes for clients that may rely on either method
-	router.Get("/v1/exporter/decideds", api.Handler(s.exporter.Decideds))
-	router.Post("/v1/exporter/decideds", api.Handler(s.exporter.Decideds))
+	if s.full {
+		// @Summary Get validator traces
+		// @Description Returns traces for a specific validator
+		// @Tags Exporter
+		// @Produce json
+		// @Success 200 {object} handlers.ValidatorTracesResponse
+		// @Router /v1/exporter/traces/validator [get]
+		router.Post("/v1/exporter/traces/validator", api.Handler(s.exporter.ValidatorTraces))
```

Comment:
just a question.
Since we call it Exporter V2, maybe the API endpoint should also have v2 prefix? Or these endpoints are new, so there is no reason to start it from v2?